blog/2015/01/future-and-funding-of-science.html

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+root: ../../..
+author: Greg Wilson
+title: "The Future and Funding of Science"
+date: 2015-01-04
+time: "05:30:00"
+category: ["Opinion"]
+---
+<!-- start excerpt -->
+<p>
+  I was talking with friends over the holiday about the future of science
+  and how it might one day be funded.
+  Since it'll be ten years before I'm proven wrong,
+  it seems like a good topic with which to start the new year.
+</p>
+<!-- end excerpt -->
+<p>
+  First,
+  I believe that <em>everything</em> in the next fifty years
+  is going to be shaped by a bunch of bills coming due at the same time.
+  Thanks to our generation's selfish dithering,
+  global climate change,
+  resource depletion,
+  drug-resistant diseases,
+  the Sixth Great Extinction,
+  and aging populations in the industrialized world
+  are all going to hit us at more or less the same time.
+  Research that doesn't directly address one or more of these problems
+  will be sidelined in favor of research that does&mdash;basically,
+  trips to Mars and LHC-style "big physics" will be starved
+  so that climatology, materials science, and ecology can be fed.
```

Comment:
Yeah, definitely need medicine here. And while materials science is the foundation of civilization, there's nothing in the text before here that explicitly relates to it.